modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -48,6 +49,8 @@
     private static final int SELENIUM_PORT = 4444;
     private static final int VNC_PORT = 5900;
 
+    private static final long DEFAULT_SHM_SIZE = 2 * FileUtils.ONE_GB;
```

Comment:
is it really necessary to allocate 2Gb to it? I assume some environments may not have that many RAM for containers 😅

Reply:
yes I agree... but that's what is recommended by selenium: https://github.com/SeleniumHQ/docker-selenium#running-the-images 
Should I set less for the default, and users set it higher if needed?

Reply:
Yes, I would recommend doing it this way 👍 

Also, I wonder if mounting the host's SHM (`-v /dev/shm:/dev/shm`) is a better choice, actually (as described in the link you sent)

Reply:
Following on from this, it sounds like in some cases that `/dev/shm` isn't enough (it looks like with Docker for Mac it's ~1GB by default). However, I suspect it's a safe default.

How about this: 
* if `shmSize` has been set by the developer, just respect that (so for memory hungry cases people can use 2GB or whatever they like, within the constraints of what their dev/build machines support)
* otherwise, if `shmSize` is not set, mount `/dev/shm`

Would there be any downsides I've not thought of?

Reply:
Sounds good 👍

Reply:
ok, I wil do that :)